eng/targets/DoNotGenerateSatelliteAssemblies_Workaround.targets

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project>
+  <!-- There is no way to disable generating satellite assemblies other than to override the target definitions -->
```

Comment:
Where do the core targets live? I don't mind this in the short term but longer term would be nice to have a `/p:GenerateSatelliteAssemblies=false` style property that we could pass around.

Reply:
The SDK defines `GenerateSatelliteAssembliesForCore` property, setting to false should disable both targets.

Reply:
note: today we use al.exe to generate satellite assemblies for VB and the sdk task for C#. I could change this logic to only override the target that is actually going to be executed but that seems needlessly complicated.

Reply:
The only alternative approach I can see other than overriding the targets is to remove all `_SatelliteAssemblyResourceInputs` items before either of these targets execute, I didn't see this as being necessarily better

Reply:
I see. Overriding the targets makes sense then. Maybe check with SDK team if they are interested in adding a better way.